### ISSUE TITLE
Developer Guidelines page missing

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/api-issue-tracker#513]
### API Documentation Issue

The Developer Center Guidelines page is still not fixed. (It has been logged internally, but not here apparently.)

The old URL: https://extensions.sketchup.com/en/developer_center/ew_developer 

An archived page at the Wayback Machine:
https://web.archive.org/web/20190427195822/http://extensions.sketchup.com/en/developer

We have many posts pointing coders to the old broken URL. **Please fix** !

**Comment by DanRathbun** [comment]
We were promised during the 2020.0 cycle that this would be fixed, yet it has not.

I keep having to point new extension devs to the wayback machine.

**Comment by DanRathbun** [comment]
I'm wondering now if perhaps the guy who "promised" might have left Trimble employ ?

Anyway I was thinking, that it seems to be  an overly difficult thing for SketchUp webpages to get updated and fixed. (I seem to remember someone saying that there is a 3rd party involved with this.)

**So, I propose that the old webpage we see in the wayback machine, get converted to markdown and reposted as part of the API extra files.** I think it would be better anyway rather than have information separated in two different places.

**Comment by thomthom** [comment]
Yea, I would prefer to have these guidelines merged into the API docs. Easier to update and I think it makes more sense to have them there. EW can link to the API doc page instead of having them hsoted there.

**Comment by DanRathbun** [comment]
HEY! Dezmo just pointed out the page has been published in the SketchUp Help Center ...
https://help.sketchup.com/en/extension-warehouse/extension-development-best-practices

It's a bit different as it does not mention or have a link to apply to be a Developer, **instead it's on it's parent page**.

Anyway ... if the Help Center is going to be where it lives, then perhaps just add a **Publishing** section and links to the bottom of the main API index page (https://ruby.sketchup.com/index.html).

Example:

> ### Publishing
> * [Developing Extensions for Extension Warehouse](https://help.sketchup.com/en/extension-warehouse/developing-extensions) 
>   * [Apply to be a Developer](https://extensions.sketchup.com/application)
>   * [Extension Signing and Encryption](https://help.sketchup.com/en/extension-warehouse/extension-encryption-and-signing)
>   * [Extension Development Best Practices](https://help.sketchup.com/en/extension-warehouse/extension-development-best-practices)
>     * [`SketchupExtension`](https://ruby.sketchup.com/SketchupExtension.html) class
>     * [`Sketchup::Licensing`](https://ruby.sketchup.com/Sketchup/Licensing.html) module
>       * [`Sketchup::Licensing::ExtensionLicense`](https://ruby.sketchup.com/Sketchup/Licensing/ExtensionLicense.html) class
> 

----
markdown:
```markdown
### Publishing
* [Developing Extensions for Extension Warehouse](https://help.sketchup.com/en/extension-warehouse/developing-extensions)
  * [Apply to be a Developer](https://extensions.sketchup.com/application)
  * [Extension Signing and Encryption](https://help.sketchup.com/en/extension-warehouse/extension-encryption-and-signing)
  * [Extension Development Best Practices](https://help.sketchup.com/en/extension-warehouse/extension-development-best-practices)
    * [`SketchupExtension`](https://ruby.sketchup.com/SketchupExtension.html) class
    * [`Sketchup::Licensing`](https://ruby.sketchup.com/Sketchup/Licensing.html) module
      * [`Sketchup::Licensing::ExtensionLicense`](https://ruby.sketchup.com/Sketchup/Licensing/ExtensionLicense.html) class
```

----

The publishing of these pages should have been (and still should) be announced in the public forum ...
* [**Developer Announcements**](https://forums.sketchup.com/c/developers/developer-announcements/67) category

... otherwise we cannot know they exist and change our bookmarks and point new devs there.

;)

**Comment by thomthom** [comment]
Wow... that was news to me...

I still think it's better to have the guidelines with the SketchUp Ruby API documentation.

**Comment by DanRathbun** [comment]
Yes, I agree also as it will get updated quicker, and the API docs are where developers will come looking for information **first**.
